RubberduckTests/Refactoring/EncapsulateField/EncapsulateFieldConflictFinderTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using NUnit.Framework;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Refactorings;
+using Rubberduck.Refactorings.EncapsulateField;
+using Rubberduck.VBEditor.SafeComWrappers;
+using RubberduckTests.Mocks;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RubberduckTests.Refactoring.EncapsulateField
+{
+    [TestFixture]
+    public class EncapsulateFieldConflictFinderTests
```

Comment:
I miss test cases for all the cases where there should still be a result. Are they in some other file?

Reply:
The conflict detection test cases where there is an expected result are exercised by every test in the `EncapsulateFieldRefactoring` test battery.  

Certainly, many `Refactoring` tests focus primarily on conflict detection and could be migrated to this class.